config/stack_noaa.yaml

```diff
@@ -25,12 +25,12 @@ szip:
 jasper:
   build: YES
   shared: NO
-  version: 2.0.25
+  version: 2.0.22
```

Comment:
please update the branch to develop.  develop is pointing to 2.0.25

Reply:
Done.